src/test/java/com/squareup/phrase/PhraseTest.java

```diff
@@ -166,4 +169,22 @@ public void formatFailsFastWhenKeysAreMissing() {
     assertThat(formatted.toString()).isEqualTo("Hello Abe, you are 20 years old.");
     assertThat(formatted).isInstanceOf(Spannable.class);
   }
+
+  @Test public void testIntoSetsTargetText() {
+    Context context = Robolectric.application;
+    TextView textView = new TextView(context);
+
+    Phrase.from("Hello {user}!").put("user", "Eric").into(textView);
+    CharSequence actual = textView.getText().toString();
+
+    assertThat(actual.toString()).isEqualTo("Hello Eric!");
+  }
+
+  @Test public void testIntoNullFailsFast() {
+    try {
+      Phrase.from("Hello {user}!").put("user", "Eric").into(null);
```

Comment:
You'll also want to fail if the exception is not thrown at all.

Ideally, all these tests should use `@Rule ExpectedException`.